puppet/zulip/files/postgresql/process_fts_updates

```diff
@@ -76,12 +76,18 @@ if remote_postgres_host != '':
     postgres_password = ''
     if settings.DATABASES['default']['PASSWORD'] is not None:
         postgres_password = "password='%s'" % settings.DATABASES['default']['PASSWORD']
+    postgres_user = 'user=zulip'
+    if settings.DATABASES['default']['USER'] is not None:
+        postgres_user = "user='%s'" % settings.DATABASES['default']['USER']
+    postgres_dbname = 'dbname=zulip'
+    if settings.DATABASES['default']['NAME'] is not None:
+        postgres_dbname = "dbname='%s'" % settings.DATABASES['default']['NAME']
```

Comment:
I think we don't need conditionals here; can't we just unconditionally base on on `settings.DATABASES`?  We only need the conditionals for password because it might be empty if we're using peer authentication.